### PR TITLE
Update About Page for v1

### LINKED ISSUE #370: Update the About Us page with new copy specifying v1 language
Right now our About page contains language describing that we are in a demo. Update the copy for language that expresses  we are launched, still need to develop remaining chapters, and invite contributors. 

Screenshot 2023-04-11 at 2 33 11 PM

### PR DESCRIPTION
**Pull request by ecurrencyhodler**
Closes #370 

We can wait on merging this until Tuesday 4/25.

## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Great catch Saba. Thanks! Added a link now.

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -32,11 +32,9 @@ const translations = {
     project: {
       title: 'Built with fun',
       paragraph_one:
-        'Saving Satoshi is a light-hearted first point of contact for developers who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image (aka AI) tools like Midjourney.',
+        'Saving Satoshi is a light-hearted first point of contact for coders who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image tools like Midjourney.',
       paragraph_two:
-        'This version of the project is currently in a demo state with a goal of getting feedback on the overall experience and providing a proof of concept for early users. In light of this, user progress is stored locally in the browser and the story along with images are not finalized.',
-      paragraph_three:
```
Comment (afterburn) [inline i18n/locales/en.ts:38]: You removed paragraph_three which has this result:

image

You should also remove the following code from `app/[lang]/about/page.tsx` on line 52:

```tsx
<p>{t('about.project.paragraph_three')}</p>
```

**Review by afterburn — CHANGES_REQUESTED** [review]
Hey @ecurrencyhodler I requested a small change.

File: i18n/locales/en.ts (hunk 1 above)
Comment (ecurrencyhodler) [reply i18n/locales/en.ts:38]: Good catch thanks! Changes made.

**Review by afterburn — APPROVED** [review]

File: i18n/locales/en.ts [hunk 2]
```diff
@@ -32,11 +32,9 @@ const translations = {
     project: {
       title: 'Built with fun',
       paragraph_one:
-        'Saving Satoshi is a light-hearted first point of contact for developers who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image (aka AI) tools like Midjourney.',
+        'Saving Satoshi is a light-hearted first point of contact for coders who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image tools like Midjourney.',
```
Comment (shaavan) [inline i18n/locales/en.ts:35]: nit:

```suggestion
        'Saving Satoshi is a light-hearted first point of contact for coders curious about Bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source, and all the art has been generated using text-to-image tools like Midjourney.',
```

File: i18n/locales/en.ts [hunk 3]
```diff
@@ -32,11 +32,9 @@ const translations = {
...
       paragraph_two:
-        'This version of the project is currently in a demo state with a goal of getting feedback on the overall experience and providing a proof of concept for early users. In light of this, user progress is stored locally in the browser and the story along with images are not finalized.',
-      paragraph_three:
-        'Based on the feedback we receive, we will then work towards an official release (we call it V1) that includes 2-3 full lesson chapters, accounts, and more. 10 chapters are planned out and will be released over time as they are completed. Additional features may include collaborative learning, badges, extra-challenging side quests, and more.',
+        'This free and open source software (FOSS) project is in continuous development. Our goal is to take an iterative approach to incorporate feedback as we release new chapters. There are currently 10 chapters mapped out, however more may be added as the project continues to grow.',
```
Comment (shaavan) [inline i18n/locales/en.ts:37]: nit:
```suggestion
        'This free and open source software (FOSS) project is in steady development. We aim to take an iterative approach to incorporate feedback as we release new chapters. There are currently ten chapters mapped out. However, more may be added as the project continues to grow.',
```

**Review by shaavan — COMMENTED** [review]
ACK

- The copy changes look great, and update the text to match the current development state.
- I have made a few nit comments that will help improve the flow of the text.

File: i18n/locales/en.ts (hunk 2 above)
Comment (ecurrencyhodler) [reply i18n/locales/en.ts]: Updated!

File: i18n/locales/en.ts (hunk 3 above)
Comment (ecurrencyhodler) [reply i18n/locales/en.ts]: Thanks for zeroing on this section. I incorporated some of your suggestions but also updated the copy over all.

**Review by shaavan — APPROVED** [review]
ACK! 🚀

**Pull request merged**